docs/learn/fundamentals/transactions/unique-source-for-operation.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: "When Operation Source Accounts differ from that of the Parent Transaction"
+sidebar_label: Unique Source Accounts for Operations
+description: "Learn"
+sidebar_position: 40
+---
+
+# When Operation Source Accounts differ from that of the Parent Transaction
+
+Let's review the unique case where an operation may have a source account different from that of the parent transaction envelope.
+
+There are 2 main situations where this may happen:
+
+1. Operations that affect more than one account
+2. Operations with a high threshold weight
+
+## Operations that Affect multiple Accounts
+
+If a transaction has operations with multiple source accounts, it will require the source account signature for each operation.
```

Comment:
this could be clearer. it reads a little like a single operation could have multiple source accounts attached to it. and `signature for each operation` sounds like every operation has to be signed. the transaction requires a valid signature from all source accounts present in the tx and/or the operations.